src/api/InkAPI.cc

```diff
@@ -4229,6 +4229,54 @@ tsapi::c::TSContScheduleOnThread(TSCont contp, TSHRTime timeout, TSEventThread e
   return action;
 }
 
+std::vector<tsapi::c::TSAction>
+tsapi::c::TSContScheduleOnEntirePool(TSCont contp, TSHRTime timeout, TSThreadPool tp)
```

Comment:
How does something in the C API return a `std::vector`?

Reply:
@bneradt said it's all c++ api now.

Reply:
The `tsapi::c` part.